armv7m7-imxrt106x/syspage.c

```diff
@@ -23,6 +23,7 @@
 #define MAX_MAPS_NB             16
 #define MAX_ARGS_SIZE           256
 #define MAX_ENTRIES_NB          6     /* 3 of kernel's sections, 2 of plo's sections and syspage */
+#define MAX_CMDLINE_SIZE        16
 
 #define MAX_SYSPAGE_SIZE     (sizeof(syspage_t) + MAX_ARGS_SIZE * sizeof(char) + MAX_PROGRAMS_NB * sizeof(syspage_program_t) + MAX_MAPS_NB * sizeof(syspage_map_t))
```

Comment:
The standard `C` probably guarantees `sizeof(char) == 1`.

Reply:
I see, but this does not apply to topic of this change (PR).